src/overlays/actors/ovl_En_Changer/z_en_changer.c

```diff
@@ -20,15 +35,257 @@ const ActorInit En_Changer_InitVars = {
     (ActorFunc)EnChanger_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Changer/EnChanger_Destroy.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Changer/EnChanger_Init.s")
+static Vec3f sLeftChestPos[] = {
+    { 0.0f, 0.0f, 0.0f },         { -100.0f, 20.0f, -245.0f },  { -100.0f, 20.0f, -685.0f },
+    { -100.0f, 20.0f, -1125.0f }, { -100.0f, 20.0f, -1565.0f }, { -100.0f, 20.0f, -2005.0f },
+};
+
+static Vec3f sRightChestPos[] = {
+    { 0.0f, 0.0f, 0.0f },        { 140.0f, 20.0f, -245.0f },  { 140.0f, 20.0f, -685.0f },
+    { 140.0f, 20.0f, -1125.0f }, { 140.0f, 20.0f, -1565.0f }, { 140.0f, 20.0f, -2005.0f },
+};
+
+static s32 sLoserGetItemIds[] = {
+    GI_NONE, GI_RUPEE_GREEN_LOSE, GI_RUPEE_GREEN_LOSE, GI_RUPEE_BLUE_LOSE, GI_RUPEE_BLUE_LOSE, GI_RUPEE_RED_LOSE,
+};
+
+static s32 sEtcParams[] = { 0x0000, 0x0008, 0x0008, 0x0009, 0x0009, 0x000A };
```

Comment:
Values from the `ItemEtceteraType` enum should probably be used here, except the 0 which corresponds to the first room so I guess it's nothing.

I also suggest naming the array `sItemEtcTypes`